### DISCUSSION TITLE
Is it possible to change the drive icon in Windows (Mountain Duck)?

### DISCUSSION THREAD

**Discussion by Ralmarax** [iterate-ch/cyberduck#14347] [Q&A]
Is it possible to change the default icon of a drive? Unfortunately, the Windows-specific registry option (eg HKCU\SOFTWARE\Classes\Applications\Explorer.exe\Drives\D\DefaultIcon)  to override the drive icon is used and overwritten by Mountain Duck. Any ideas?
Best regards

**Comment by AliveDevil** [comment]
We don't have a way of specifically setting a drive icon for Mountain Duck and decided, that using the icon of the bookmark protocol is a sufficient way of conveying that those drives are created from Mountain Duck, while also including the type of connection (FTP, S3, etc.)

Can you elaborate on your use case, so we can assess your use-case better?

**Reply by Ralmarax** [reply]
I have several (WebDav) drives, which all use the same icon. For file operations, I very often use TotalCommander, which has a drive toolbar with smaller icons in comparison with the windows explorer.
Therefore I wanted to use the DefaultIcon registry key to change the icons for better identify the different drives. Unfortunately, I found that this windows registry key is overwritten/deleted by Mountain Duck and therefore not usable for my purposes.
